packages/support/src/helpers.php

```diff
@@ -181,6 +181,12 @@ function generate_search_column_expression(string $column, ?bool $isSearchForced
         };
 
         if ($isSearchForcedCaseInsensitive) {
+            if (in_array($driverName, ['mysql', 'mariadb'], true) && str_contains($column, '->')) {
```

Comment:
If the column already contains `json_extract()` (as per the check above), do we still need to do this?

Reply:
I am not quite sure, in what situation would there be `json_extract(` and `->` in the column? I presumed either one of them would be present, as `->` is a short hand/alias for `json_extract(`

Reply:
I changed it to exclude the situation when the column already starts with `json_extract(`, which is probably the most careful solution

Reply:
`json_extract(` is added by the translatable plugin for translatable fields before being passed to this function